examples/fromtf/upload.py

```diff
@@ -0,0 +1,53 @@
+import argparse
+import os
+import pickle
+import numpy as np
+from PIL import Image
+
+from hub.collections import dataset, tensor
+
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+
+def is_equal(lds, rds):
+    output = True
+    for lex, rex in zip(lds, rds):
+        comparsion = lex['data'].numpy() == rex['data'].numpy()
+        output *= comparsion.all()
+    return output
+
+
+def check_one(name):
```

Comment:
This should go as test_[python_file_name].py file near the function implemented.